### PR TITLE
Adding cleanup for etcd, docker state

### PR DESCRIPTION
**Pull request by neelimamukiri**
Also making the docker image smaller and moving out the ansible parts to the tgz file to allow easier customization.
Fixes https://github.com/contiv/install/issues/12
Corresponding ansible PR is https://github.com/contiv/ansible/issues/320

### CODE REVIEW

File: cluster/Vagrantfile [hunk 1]
```diff
@@ -96,7 +96,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           c.vm.provision :shell, path: 'bootstrap_centos.sh'
           if role == 'master'
             # Install kubernetes on master
-            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.4.7'
+            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.5.2'
```
Comment (rhim) [inline cluster/Vagrantfile]: Why is this needed? Should we not stick to same version as supported matrix for now?

Comment (neelimamukiri) [inline cluster/Vagrantfile]: Good point, its just that kubeadm users are likely going to keep moving on faster. I will change it back to 1.4.7 for now and we can discuss version support policies etc.
